puzzles/templatetags/puzzle_extras.py

```diff
@@ -35,6 +35,8 @@ def get_title(puzzle):
 def puzzle_class(puzzle):
     if puzzle.status == Puzzle.PENDING:
         return "table-warning"
+    elif puzzle.status == Puzzle.EXTRACTION:
+        return "table-danger"
```

Comment:
What do you think about using yellow (table-warning) for EXTRACTION and STUCK and something like blue for PENDING?

Reply:
Sure, I dont really have any strong opinions about the styling choices.